packages/core/lib/commands/init/initSource/contracts/Migrations.sol

```diff
@@ -1,16 +1,16 @@
 // SPDX-License-Identifier: MIT
-pragma solidity >=0.4.21 <0.7.0;
+pragma solidity >=0.4.22 <0.8.0;
 
 contract Migrations {
-  address public owner;
+  address public owner = msg.sender;
```

Comment:
this syntax works on 0.4.22?

Reply:
It sure does, I tested it to be sure!

Reply:
Indeed, checking the docs, it's legal all the way back to at least 0.2.1.  I can't find a way to test things that old right now, but going by release notes I suspect it was legal even as far back as 0.1.2, the oldest version out there.

Reply:
cool!